### DISCUSSION TITLE
开启打印模式，加入的空白页也计算了页码数

### DISCUSSION THREAD

**Discussion by DongdongGui** [nuaatug/nuaathesis#119] [Q&A]
开启的模式为： openright,blankleft,twoside
在正文部分如果一章结束是单页，会自动插入一页空白页，但是下一页页码数增加了。
如我上一章节结束页页码是73，插入了一页空白页，下一章开始页页码变成了75（应该是74）

**Comment by yzwduck** [comment]
空白页是要计算页码的, 这样才能做到装订后左侧是偶数页, 右侧是奇数页.
如果不计算空白页的页码, 就无法保证上述性质.
